src/PactNet/Verifier/IPactBrokerOptions.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+
+namespace PactNet.Verifier
+{
+    /// <summary>
+    /// Pact broker options
+    /// </summary>
+    public interface IPactBrokerOptions
+    {
+        /// <summary>
+        /// Use Basic authentication with the Pact Broker
+        /// </summary>
+        /// <param name="username">Pact broker username</param>
+        /// <param name="password">Pact broker password</param>
+        /// <returns>Fluent builder</returns>
+        IPactBrokerOptions BasicAuthentication(string username, string password);
+
+        /// <summary>
+        /// Use Token authentication with the Pact Broker
+        /// </summary>
+        /// <param name="token">Auth token</param>
+        /// <returns>Fluent builder</returns>
+        IPactBrokerOptions TokenAuthentication(string token);
+
+        /// <summary>
+        /// Enable pending pacts
+        /// </summary>
+        /// <returns>Fluent builder</returns>
+        IPactBrokerOptions EnablePending();
+
+        /// <summary>
+        /// Consumer tag versions to retrieve
+        /// </summary>
+        /// <param name="tags">Consumer tags</param>
+        /// <returns>Fluent builder</returns>
+        IPactBrokerOptions ConsumerTags(params string[] tags);
+
+        /// <summary>
+        /// Include WIP pacts since the given date
+        /// </summary>
+        /// <param name="date">WIP cut-off date</param>
+        /// <returns>Fluent builder</returns>
+        IPactBrokerOptions IncludeWipPactsSince(string date); // TODO: Should this be a DateTime or DateTimeOffset so we can format it properly?
```

Comment:
In answer of your TODO. I think it should be and the implementation would format it correctly.

Reply:
I think the problem is that it supports relative arguments like "5d" for 5 days ago, so a DateTime would prevent that. I'd have to check the CLI args though